### ISSUE TITLE
the performance of the network

### ISSUE THREAD

**Issue by duxiangcheng** [TencentYoutuResearch/ImageColorization-ColorFormer#2]
Thank you for the source code. I am confused about the performance, 
When I trained the network, the colorfulness score was only 32.51, much lower than the value of the paper.
Did you use tricks during training?
Besides, I also apply your pre-trained model, the metric is also lower than the paper.
